src/test/java/capstone/mfslbackend/service/TeamServiceTest.java

```diff
@@ -97,7 +113,11 @@ public void testCreateTeamsInLeague_SuccessExists() throws IOException {
         Mockito.when(apiService.getRequest(Mockito.any(), Mockito.eq(TeamsContainer.class)))
                 .thenReturn(teamsContainer);
         Long newTeamId = 2L;
-        Team team = new Team(newTeamId, teamName, teamName, new ArrayList<>());
```

Comment:
can you keep the code consistent. There are times you are getting rid of uses of a constructor and setting attributes and other times where you are getting rid of set attributes and using a constructor